test/functional/currency/currency-formatter.js

```diff
@@ -45,7 +45,9 @@ QUnit.module( ".currencyFormatter( currency [, options] )", {
 		});
 		Globalize.locale( "en" );
 	},
-	teardown: util.resetCldrContent
+	teardown: function() {
+		util.resetCldrContent();
```

Comment:
Indeed, this is a preparation for a Cache change (https://github.com/jquery/globalize/commit/76f16d605ba904510f9aed9b58699d1c78cd22b0#diff-7762adfa36f42583983ea0b59eb78b71R49). So, this should live in #470.